### PR TITLE
Readonly always for linked_project

### PR DESCRIPTION
**Pull request by rubencabrera**
As discussed in #146

### CODE REVIEW

**Review by elicoidal — APPROVED** [review]

**Comment by elicoidal** [comment]
@rubencabrera Can you please check travis?

**Comment by rubencabrera** [comment]
Sure, I'm onto it, seems to be something to do with the product_id onchange, seems related to the recently merged PR, but it was solved there. Let me check.

**Comment by rubencabrera** [comment]
The test was right, that product_id onchange should still change the description (name field). The mentioned pr was for the deliverable, not the resource. So this last commit also fixes an error of PR #132

**Comment by rubencabrera** [comment]
Ok, the travis fails because a test of business_requirement_deliverable_resource, I was going to change it but I'm not sure which is the expected behaviour.
The test calls product_id_change and asserts that the resource name is equal to the product name, but the function doesn't do that. Should we change it? To do exactly what?

**Comment by elicoidal** [comment]
Not sure either...
@victormartinelicocorp could you have a quick look?

File: business_requirement_deliverable/models/business.py [hunk 1]
```diff
@@ -78,8 +78,6 @@ def product_id_change(self):
             uom_id = product.uom_id.id
         if product.description_sale:
             description += '\n' + product.description_sale
-        if not self.name:
```
Comment (elicoidal) [inline business_requirement_deliverable/models/business.py]: @rubencabrera 
After reconsidering: why should we have this change in this PR?
I think you need to separate this modification that was done in another PR (may rebase)?
@victormartinelicocorp @darshan-serpent

**Review by elicoidal — CHANGES_REQUESTED** [review]

File: business_requirement_deliverable/models/business.py (hunk 1 above)
Comment (serpentcs-dev1) [reply business_requirement_deliverable/models/business.py]: @rubencabrera As you know the code for onchange on the resource is already merged in #132 
and the test cases to support it were already added in Open PR #139 check this commit
https://github.com/OCA/business-requirement/pull/139/commits/2a02aedc74743772bff01b9e178b4408f01eae1f

So the travis is already fixed in the above mentioned PR.

So IMO you just need to revert your last commit and that will do all.

cc @elicoidal @victormartinelicocorp

Comment (rubencabrera) [reply business_requirement_deliverable/models/business.py]: That's what I thought, just wanted to get rid of the annoying travis failing for outside code. I'll revert and commit ASAP. Thanks again to all of you for your input.

**Comment by darshan-serpent** [comment]
+1

File: business_requirement_deliverable/models/business.py (hunk 1 above)
Comment (elicoidal) [reply business_requirement_deliverable/models/business.py]: @rubencabrera any news?

**Review by elicoidal — APPROVED** [review]

**Comment by elicoidal** [comment]
Thanks!

**Pull request merged**